src/Compilers/CSharp/Test/Symbol/Symbols/IndexedPropertyTests.cs

```diff
@@ -412,22 +412,22 @@ .maxstack  3
   // Code size       33 (0x21)
   .maxstack  4
   .locals init (IA V_0,
-  int V_1,
-  int V_2)
+                int V_1,
```

Comment:
The changes in this file are just slot order changes: the resulting effect is unchanged, we just use the earlier slot before the later slot now.